README.md

```diff
@@ -9,3 +9,29 @@ By [opening an issue](https://github.com/tidb-challenge-program/register/issues/
 ## Points Inheritance
 
 To inherit the points you earned in previous seasons, you must sign up by the same name with what you used back then. This will be reflected as your Accumulative Points. If it's over 200, you will direcly skip the Easy level tasks. If you run into any issues inheriting historical points, contact @you06 to help you out.
```

Comment:
```suggestion
To inherit the points you earned in previous seasons, you must sign up by the same name with what you used back then. This will be reflected as your Accumulative Points. If it's over 200, you can directly skip the Easy level tasks. If you run into any issues inheriting historical points, contact @you06 to help you out.
```

Reply:
As out strategy, with more than 200 points in a repo, player cannot pick more easy tasks under this repo anymore. We've discussed about it in [last PR](https://github.com/tidb-challenge-program/register/pull/32#discussion_r389237253). IMO we might use "we will" instead of "you can" here.